<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QWidget" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SplashMol</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/assets/assets/SplashMol.png</normaloff>:/assets/assets/SplashMol.png</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Type in your formula here......</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calcButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>113</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate!</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Result</string>
   </property>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>&lt;b&gt; See result here &lt;/b&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
